examples/games/desktoy.rs

```diff
@@ -0,0 +1,320 @@
+//! Bevy logo as a desk toy! Now with Googly Eyes!
```

Comment:
We can add more meat here, laying out the various features used. Drag-and-drop, transparent windows, animation are the big ones.